modules/options-for-notebook-server-environments.adoc

```diff
@@ -351,7 +351,7 @@ a| * Python 3.9
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[https://rstudio.org/] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
```

Comment:
Same as first comment.

Reply:
Updated to remove - also applied to all other instances of 'should' in the disclaimers.